fem/eltrans.cpp

```diff
@@ -200,24 +200,65 @@ void IsoparametricTransformation::Transform (const DenseMatrix &matrix,
    }
 }
 
+const IntegrationPoint IsoparametricTransformation::FindClosestRefinedPoint(
+   const Vector& pt, int order)
+{
+   const int geom = FElem->GetGeomType();
+
+   // Deal with default and invalid values for order parameter
+   int refOrder = (order < 0) ? FElem->GetOrder() : order;
```

Comment:
@tzanio -- I changed the behavior from my original PR
* -1 is the default, which uses ``FElem->GetOrder()``, as requested
* 0 now gives the element center rather than ``RefinedIntRules`` with order 0, which would give a 2x2 grid (in the quadrilateral case).

I documented this, and think it should give the desired behavior.